CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+```
+cp test.env.example test.env
+```
+to create a blank test.env to fill in.
+
+> In general, it's most important to have successful unit and Postgres tests. Once you open a PR, `dbt-core` will automatically run integration tests for the other three core database adapters. Of course, if you are a BigQuery user, contributing a BigQuery-only feature, it's important to run BigQuery tests as well.
+
+## Testing
+
+`dbt-core` uses test credentials specified in a `test.env` file in the root of the repository for non-Postgres databases. This `test.env` file is git-ignored, but please be _extra_ careful to never check in credentials or other sensitive information when developing against `dbt-core`. To create your `test.env` file, copy the provided sample file, then supply your relevant credentials. This step is only required to use non-Postgres databases.
+
+```
+cp test.env.sample test.env
+$EDITOR test.env
+```
+
+> In general, it's most important to have successful unit and Postgres tests. Once you open a PR, `dbt-core` will automatically run integration tests for the other three core database adapters. Of course, if you are a BigQuery user, contributing a BigQuery-only feature, it's important to run BigQuery tests as well.
+
+### Test commands
+There are a few mehtods for running tests locally.
+
+####`tox`
+`tox` takes care of managing virtualenvs and install dependencies in order to run tests. You can also run tests in parallel, for example you can run unit tests for Python 3.7, Python 3.8, Python 3.9 `flake8` checks in parallel with `tox -p`. Also you  can run unit tests for specific python versions with `tox -e py37`. The configuration of theses tests are located in `tox.ini`.
+
+####`pyteest`
+Finally, you can also run  a specifiic test or group of tests using `pytest` directlly. With virtualenv activa and dev dependencies installed you can do things like:
+```sh
+# run specific postgres integration tests
+python -m pytest -m profile_bigquery test/integration/001_simple_copy_test
+# run all unit tests in a file
+python -m pytest test/unit/test_bigquery_adapter.py
+# run a specific unit test
+python -m pytest test/unit/test_bigquery_adapter.py::TestBigQueryAdapter::test_copy_table_materialization_table
+```
+## Updating Docs
+
+Many changes will require and update to the `dbt-bigquery` docs here are some useful resources.
+
+- Docs are [here](https://docs.getdbt.com/).
+- The docs repo for making changes is located [here]( https://github.com/dbt-labs/docs.getdbt.com).
+- The changes made are likely to impact one or both of [BigQuery profile](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile), or [BigQuery configs](https://docs.getdbt.com/reference/resource-configs/bigquery-configs).
+- We as every community member who makes a user-facing change to open and issue for their change in that repo.
+- If the contributor feels up to they should contribute to the PR for the docs update as well.
```

Comment:
I think you can remove these lines. Or suggest creating an issue or a PR for docs changes.